### DISCUSSION TITLE
Generated styles not applying

### DISCUSSION THREAD

**Discussion by nelsonmic** [tailwindlabs/tailwindcss#2419] [Help]
Hello there, so I finish setting tailwind for my project, and for some reason the style classes doesn't seem to apply to HTML elements, I've been trying to figure out what the problem is but nothing is working, the first problem I had was with the @tailwind directives, VS code kept telling me "unknown at rules" i was able to fix that by changing its settings to ignore, but now after generating the css file, VS code tells me there's 1 problem with generated css file, can someone help me out

**Comment by simonswiss** [comment]
Hi @theSultan00 👋

Would love to help, but going to need more details than that. Is it possible that you share a reproduction of your code on a GitHub repo or CodeSandbox?

Can you see a generated CSS file with the Tailwind utility classes?

To silence the vscode warnings about unknowns @ rules, you can add a `postcss` language support extension and change the language for your CSS file to be `postcss`.

What message are you getting for the `1 problem with generated CSS file`?
